<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
       <width>1303</width>
       <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                      <widget class="QLabel" name="label_find">
                          <property name="sizePolicy">
                              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                  <horstretch>0</horstretch>
                                  <verstretch>0</verstretch>
                              </sizepolicy>
                          </property>
                          <property name="text">
                              <string>Find:</string>
                          </property>
                      </widget>
                  </item>
                  <item>
                      <widget class="QLineEdit" name="lineEdit_string">
                          <property name="sizePolicy">
                              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                  <horstretch>0</horstretch>
                                  <verstretch>0</verstretch>
                              </sizepolicy>
                          </property>
                      </widget>
                  </item>
                  <item>
                      <spacer name="horizontalSpacer">
                          <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeType">
                              <enum>QSizePolicy::Expanding</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                              <size>
                                  <width>40</width>
                                  <height>20</height>
                              </size>
                          </property>
                      </spacer>
                  </item>
              </layout>
          </item>
          <item>
              <widget class="QSplitter" name="splitter">
                  <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                      </sizepolicy>
                  </property>
                  <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                  </property>
                  <widget class="QTreeWidget" name="treeWidget">
                      <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                              <horstretch>0</horstretch>
                              <verstretch>0</verstretch>
                          </sizepolicy>
                      </property>
                      <attribute name="headerMinimumSectionSize">
                          <number>100</number>
                      </attribute>
                      <attribute name="headerStretchLastSection">
                          <bool>false</bool>
                      </attribute>
                      <column>
                          <property name="text">
                              <string>#</string>
                          </property>
                      </column>
                      <column>
                          <property name="text">
                              <string>Filename</string>
                          </property>
                      </column>
                      <column>
                          <property name="text">
                              <string>FilePath</string>
                          </property>
                      </column>
                      <column>
                          <property name="text">
                              <string>pathtofile</string>
                          </property>
                      </column>
                  </widget>
                  <widget class="QWidget" name="">
                      <layout class="QVBoxLayout" name="verticalLayout">
                          <item>
                              <layout class="QHBoxLayout" name="horizontalLayout_2">
                                  <item>
                                      <widget class="QLabel" name="label_matches">
                                          <property name="text">
                                              <string>Matches:</string>
                                          </property>
                                      </widget>
                                  </item>
                                  <item>
                                      <widget class="QLabel" name="label_matches_count">
                                          <property name="text">
                                              <string>#Matches</string>
                                          </property>
                                      </widget>
                                  </item>
                                  <item>
                                      <spacer name="horizontalSpacer_2">
                                          <property name="orientation">
                                              <enum>Qt::Horizontal</enum>
                                          </property>
                                          <property name="sizeHint" stdset="0">
                                              <size>
                                                  <width>40</width>
                                                  <height>20</height>
                                              </size>
                                          </property>
                                      </spacer>
                                  </item>
                                  <item>
                                      <widget class="QPushButton" name="pushButton_previous">
                                          <property name="text">
                                              <string>&lt;&lt;</string>
                                          </property>
                                      </widget>
                                  </item>
                                  <item>
                                      <widget class="QPushButton" name="pushButton_next">
                                          <property name="text">
                                              <string>&gt;&gt;</string>
                                          </property>
                                      </widget>
                                  </item>
                              </layout>
                          </item>
                          <item>
                              <widget class="QTextEdit" name="textEdit"/>
                          </item>
                          <item>
                              <layout class="QHBoxLayout" name="horizontalLayout_4">
                                  <item>
                                      <widget class="QPushButton" name="pushButton_open_path">
                                          <property name="text">
                                              <string>Open path</string>
                                          </property>
                                      </widget>
                                  </item>
                                  <item>
                                      <widget class="QPushButton" name="pushButton_open_file">
                                          <property name="text">
                                              <string>Open file</string>
                                          </property>
                                      </widget>
                                  </item>
                              </layout>
                          </item>
                      </layout>
                  </widget>
              </widget>
          </item>
          <item>
              <layout class="QGridLayout" name="gridLayout">
                  <item row="2" column="1">
                      <widget class="QProgressBar" name="progressBar_searching">
                          <property name="value">
                              <number>24</number>
                          </property>
                      </widget>
                  </item>
                  <item row="1" column="2">
                      <widget class="QLabel" name="label_indexing_time">
                          <property name="text">
                              <string>indexing_time</string>
                          </property>
                      </widget>
                  </item>
                  <item row="1" column="1">
                      <widget class="QProgressBar" name="progressBar_indexing">
                          <property name="value">
                              <number>24</number>
                          </property>
                      </widget>
                  </item>
                  <item row="2" column="0">
                      <widget class="QLabel" name="label_searching">
                          <property name="text">
                              <string>Searching:</string>
                          </property>
                      </widget>
                  </item>
                  <item row="2" column="2">
                      <widget class="QLabel" name="label_searching_time">
                          <property name="text">
                              <string>searching_time</string>
                          </property>
                      </widget>
                  </item>
                  <item row="0" column="2">
                      <widget class="QLabel" name="label_time">
                          <property name="text">
                              <string>Time:</string>
                          </property>
                      </widget>
                  </item>
                  <item row="1" column="0">
                      <widget class="QLabel" name="label_indexing">
                          <property name="text">
                              <string>Indexing:</string>
                          </property>
                      </widget>
                  </item>
                  <item row="0" column="1">
                      <widget class="QLabel" name="label_progress">
                          <property name="text">
                              <string>Progress:</string>
                          </property>
                      </widget>
                  </item>
              </layout>
          </item>
          <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                      <widget class="QPushButton" name="pushButton_stop_indexing">
                          <property name="text">
                              <string>Stop indexing</string>
                          </property>
                      </widget>
                  </item>
                  <item>
                      <widget class="QPushButton" name="pushButton_stop_searching">
                          <property name="text">
                              <string>Stop searching</string>
                          </property>
                      </widget>
                  </item>
              </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
        <width>1303</width>
        <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionScan_Directory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScan_Directory"/>
  </widget>
  <action name="actionScan_Directory">
   <property name="text">
    <string>&amp;Scan Directory...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
